catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswSource.java

```diff
@@ -1645,6 +1686,237 @@ public void setMetacardTypes(List<MetacardType> types) {
     }
 
     /**
+     * If a source is flagged as registry type, use the following methods for CRUD
```

Comment:
This was a leftover comment from when work first started on this ticket. I removed the comment. We have worked to keep anything registry specific out of the AbstractCswSource. The changes in this file should only be for implementing the catalog store and support for the tags concept.